src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/Microsoft.CommonLanguageServerProtocol.Framework.csproj

```diff
@@ -14,6 +14,8 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="StreamJsonRpc" Version="$(StreamJsonRpcVersion)" />
+    <!--Workaround because StreamJsonRpc is not moved to .net8, after it updates this should be removed-->
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVersion)" />
```

Comment:
💭 I don't understand the comment here since the project targets netstandard2.0

Reply:
I see this without the directly reference to `Bcl.AsyncInterfaces`
<img width="442" alt="image" src="https://github.com/dotnet/roslyn/assets/24360909/57e530a0-05d9-4c1d-9a26-87573e9a7f5a">
And with the reference:
<img width="461" alt="image" src="https://github.com/dotnet/roslyn/assets/24360909/19dff977-bc99-4d5a-9b0c-9c97bdb3064f">

And it just resolve all the build warnings

Reply:
> This is the only place we need to remember once streamrpc get updated

Moving to NuGet Central Package Management would eliminate the need to keep track of the StreamJsonRpc update here.

Reply:
While CPM helps with lifting transitives, in this case, M.Bcl.AsyncInterfaces shouldn't be coming in as a dependency at all. Once https://github.com/microsoft/vs-streamjsonrpc/commit/96b9541aaf2696b77a32522fb0387da47dacae8e is consumed (I think this will be released as part of VS 17.10) you can remove this workaround.

Reply:
FYI @tmat already has a PR to move us to CPM in main
https://github.com/dotnet/roslyn/pull/71111

Reply:
Yes, moving to CPM should definitely happen in a separate PR. That requires a big change for a repo this size.